docs/usage/help.rst.inc

```diff
@@ -32,6 +33,10 @@ Repository (or Archive) URLs and --prefix values support these placeholders:
 
     The current process ID.
 
+{borgversion}
+
+    The version of borg.
+
```

Comment:
When you make a new PR against 1.0-maint, please bear in mind that

> IMPORTANT: this file is auto-generated from borg's built-in help, do not edit!

Ie. these changes should go in the docstrings in borg.archiver